prototype_source/pt2e_quant_x86_inductor.rst

```diff
@@ -15,14 +16,15 @@ Introduction
 This tutorial introduces the steps for utilizing the PyTorch 2 Export Quantization flow to generate a quantized model customized
 for the x86 inductor backend and explains how to lower the quantized model into the inductor.
 
-The new quantization 2 flow uses the PT2 Export to capture the model into a graph and perform quantization transformations on top of the ATen graph. This approach is expected to have significantly higher model coverage, better programmability, and a simplified UX.
+The new quantization 2 flow uses the PT2 Export to capture the model into a graph and perform quantization transformations on top of the ATen graph.
```

Comment:
nit: we don't use `quantization 2 flow`, please use `pytorch 2 export quantization flow` instead

Reply:
Thanks and changed.